src/main/java/org/candlepin/subscriptions/ApplicationConfiguration.java

```diff
@@ -133,7 +133,7 @@ public static BeanFactoryPostProcessor servletInitializer() {
     @Bean
     public AccountListSource accountListSource(ApplicationProperties applicationProperties,
         AccountConfigRepository accountConfigRepository, ApplicationClock clock) {
-        if (applicationProperties.getAccountListResourceLocation() != null) {
+        if (!applicationProperties.getAccountListResourceLocation().isEmpty()) {
```

Comment:
Won't this throw a NullPointerException if not set? Perhaps we should check the null case as well?

Reply:
Fair enough, switched to `StringUtils.hasText` accordingly.